### PR TITLE
upgrade to dbt-vertica to 1.3.0 version basic setup details

### PR DESCRIPTION
**Pull request by Sachin-Thakur**
## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/sidebars.js [hunk 1]
```diff
@@ -433,6 +433,7 @@ const sidebarSettings = {
         "reference/resource-configs/azuresynapse-configs",
         "reference/resource-configs/greenplum-configs",
         "reference/resource-configs/impala-configs",
+        "reference/resource-configs/vertica-configs",
         "reference/resource-configs/doris-configs",
```
Comment (dataders) [inline website/sidebars.js]:
```suggestion
        "reference/resource-configs/doris-configs"
```

File: website/docs/reference/resource-configs/vertica-configs.md [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+<File name='vertica_incremental.sql'>
+
+```sql
+{{  config(  materialized='table',   no_segmentation='True'  ) }} 
+select * from  public.product_dimension
+
+```
+</File>
+</TabItem>
+
```
Comment (matthewshaver) [inline website/docs/reference/resource-configs/vertica-configs.md:492]: </tabs>

Comment (matthewshaver) [inline website/docs/reference/resource-configs/vertica-configs.md]:
```suggestion
</tabs>
```

File: website/docs/reference/resource-configs/vertica-configs.md [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+<File name='vertica_incremental.sql'>
+
+```sql
+{{  config(  materialized='table',   no_segmentation='True'  ) }} 
+select * from  public.product_dimension
+
+```
+</File>
+</TabItem>
+</tabs>
```
Comment (matthewshaver) [inline website/docs/reference/resource-configs/vertica-configs.md]:
```suggestion
</Tabs>
```

**Review by dataders — APPROVED** [review]
thanks @Sachin-Thakur !

**Pull request merged**